tests/controller/mockData.ts

```diff
@@ -149,7 +149,51 @@ export const mockData = {
             mint_address: "",
           },
         ],
-
+        displayActivities: {
```

Comment:
Solana backend does not have this data.
displayActivity should be constructed from preference controller which should not be mocked